File: CMS/src/app/shared/ATTPTiepNhanDocuments.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ATTPTiepNhanDocuments } from 'src/app/shared/ATTPTiepNhanDocuments.model';
import { BaseService } from './Base.service';
@Injectable({
  providedIn: 'root'
})
export class ATTPTiepNhanDocumentsService extends BaseService {
  List: ATTPTiepNhanDocuments[] | undefined;
  ListFilter: ATTPTiepNhanDocuments[] | undefined;
  FormData!: ATTPTiepNhanDocuments;
  constructor(public httpClient: HttpClient) {
    super(httpClient);
    this.Controller = "ATTPTiepNhanDocuments";
  }
}

File: Service/Implement/ATTPTiepNhanDocumentsService.cs
namespace Service.Implement
{
    public class ATTPTiepNhanDocumentsService : BaseService<ATTPTiepNhanDocuments, IATTPTiepNhanDocumentsRepository>
    , IATTPTiepNhanDocumentsService
    {
    private readonly IATTPTiepNhanDocumentsRepository _ATTPTiepNhanDocumentsRepository;
    public ATTPTiepNhanDocumentsService(IATTPTiepNhanDocumentsRepository ATTPTiepNhanDocumentsRepository) : base(ATTPTiepNhanDocumentsRepository)
    {
    _ATTPTiepNhanDocumentsRepository = ATTPTiepNhanDocumentsRepository;
    }
    }
    }

